wildfly-builder-image/tests/features/legacy-s2i.feature

```diff
@@ -109,4 +109,22 @@ Scenario: Test external driver created during s2i.
    ### PLACEHOLDER FOR CLOUD CUSTOM TESTING ###
    Then container log should contain WFLYSRV0010: Deployed "App1.war"
    Then container log should contain WFLYSRV0010: Deployed "App2.war"
-   Then container log should contain WFLYSRV0025
\ No newline at end of file
+   Then container log should contain WFLYSRV0025
+
+  Scenario: Test WildFly Maven Plugin version automatic update
+    Given s2i build https://github.com/wildfly/wildfly-s2i from test/test-app-binary with env and True using legacy-s2i-images
+      | variable                             | value         |
+      | GALLEON_PROVISION_LAYERS | jaxrs-server |
+      | GALLEON_PROVISION_FEATURE_PACKS | org.wildfly:wildfly-galleon-pack:34.0.0.Beta1, org.wildfly.cloud:wildfly-cloud-galleon-pack:7.0.2.Final |
```

Comment:
I wonder, how those versions are maintained in time. We have already 34.0.0.Final

Reply:
Yeah, we need to update to FInal, will be done in other PR.